### PR TITLE
Documentation for mass deployment.

### PR DESCRIPTION
**Pull request by allexzander**

### CODE REVIEW

File: doc/massdeploymentcommandline.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The following parameters are supported:
+
+``--userid``
+        (required) userId (username as on the server) to pass when creating an account via command-line.
+
+``--apppassword``
+        (required) appPassword to pass when creating an account via command-line (see the ``login-flow`` section in server documentation on how to generate the app password).
+
+``--localdirpath``
+        (optional) path where to create a local sync folder when creating an account via command-line. If skipped, then default local sync folder path (/home/Nextcloud<n>) will be generated by desktop client.
```
Comment (mgallien) [inline doc/massdeploymentcommandline.rst:16]: `/home/Nextcloud<n>` is not very descriptive
on Windows that would be something else and on Linux that would usually be `/home/<userid>/Nextcloud`

**Comment by allexzander** [comment]
> thanks good for me but we miss the description of `--overrideserverurl` and `--overridelocaldir` this is especially important because one can wonder what to use between `--serverurl` and `--overrideserverurl`

FIXED

File: doc/massdeploymentcommandline.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        (optional) path where to create a local sync folder when creating an account via command-line. If skipped, then default local sync folder path (/home/<userid>/Nextcloud<n> for Linux/mac or C:/<userid>/Nextcloud<n> for Windows) will be generated by desktop client.
+        
+``--isvfsenabled``
+        (optional) whether to set a VFS or non-VFS folder (1 for 'yes' or 0 for 'no') when creating an account via command-line. Default is 0.
+
+``--remotedirpath``
+        (optional) path to a remote subfolder when creating an account via command-line. e.g. If the server has folders "/Photos", "/Documents", "/Music" you can pass "/Music" and then this folder will get set up as remote root.
+
+``--serverurl``
+        (required) a server URL to use when creating an account via command-line. (NOTE: There is another parameter supported by Nextcloud desktop client ``--overrideserverurl`` but it SHOULD NOT be used here as it is intended for setup via UI with wizard)
```
Comment (mgallien) [inline doc/massdeploymentcommandline.rst:25]: can you also add the same for `--overridelocaldir` ?

**Review by mgallien — CHANGES_REQUESTED** [review]
see my comment

**Comment by allexzander** [comment]
> see my comment

fixed

File: doc/massdeploymentcommandline.rst (hunk 2 above)
Comment (allexzander) [reply doc/massdeploymentcommandline.rst:25]: @mgallien done in `doc/wizardaccountsetupcommandline.rst`

**Review by mgallien — APPROVED** [review]

**Pull request merged**